SplitImage/test/new.js

```diff
@@ -3,11 +3,47 @@
  * This spec is managed by the Jasmine testing library.
  **/
  
+
 describe ("SplitImage New Suite", function() {
   
+  var new_fixtures = [
+    '<a class="home_domain">Privly</a>',
+    '<a class="legal_nav">Legal</a>',
+    '<a class="account_url">Account<a/>',
+    '<a id="logout_link">Logout</a>',
+    '<ul><li class="mobile_hide"></li></ul>',
+    '<div id="drop_zone"></div>',
+    '<input type="file" id="files" multiple />',
+    '<button id="save"></button>'
+  ]
+
+  beforeEach(function() {
+    for (var i=0; i<new_fixtures.length; i++) {
+      document.body.innerHTML += new_fixtures[i];
+    }
+  });
+
+  afterEach(function() {
+    document.body.innerHTML = '';
+  });
+
   it("initializes properly", function() {
-    // pass
+    // todo, this should be called automatically but I don't believe
+    // the DOMContentLoaded event is being fired in this testing environment
```

Comment:
Okay. Great! I tried firing the DOMContentLoaded event manually here, just to test, and tests started failing (obviously because karma local URL was not same as home_domain url. As for the TODO, I had copied over code from Message but forgot to modify the comments. Will do this now.